docs/image_field.md

```diff
@@ -0,0 +1,121 @@
+# Image Field
+
+For the ImageField to work it's necessary to have the Pillow library installed.
+
+## Django Models
+
+The setup of the model containing the image field can be done in the following way.
+
+````python
+class BookInformation(models.Model):
+    #...
+    cover_image = models.ImageField(upload_to="covers/", null=True)
+    #...
+````
+Note that the `upload_to` option corresponds to the path in which the images are stored/saved in the media folder.
+In this case the path will be `.../library_project/media/covers/`.
+
+## Factory Boy mock data
+
+The easiest way to generate mock images data with factory boy is with the `factory.django.ImageField()` method just like the example below.
+
+```` python 
+class BookInformationFactory(factory.django.DjangoModelFactory):
+    #...
+    cover_image = factory.django.ImageField(color=factory.Faker("color"))
+    #...
+````
+
+Note that the `color` option can Hardcoded with something like `color="blue"` and that will result in the 
+generation of images with the color blue. In this case, `factory.Faker("color")` will generate random color names
+which will result in the generation of images with diferent colors for each `cover_image`.
+
+## ImageField Views Tests
```

Comment:
The view and the serializer must be presented as well, before moving on to the api test